src/main/scripts/install.sh

```diff
@@ -112,7 +112,7 @@ adminPassword=$2
 enableDB=$3
 dbType=$4
 jdbcDSJNDIName=$5
-dsConnectionURL=$6
+dsConnectionString=$6
```

Comment:
I am aware you know this, but please remember this query. https://devdiv.visualstudio.com/DevDiv/_queries/query/2625d14f-0deb-4763-b4f5-e0495204c329/

Reply:
Good catch. However, env vars are supported by `Microsoft.Resources/deploymentScripts` but not supported for `Microsoft.Compute/virtualMachines/extensions` as for now.